src/service/activity-impl.js

```diff
@@ -204,11 +209,12 @@ export class Activity {
   /** @private */
   setUpActivityListeners_() {
     for (let i = 0; i < ACTIVE_EVENT_TYPES.length; i++) {
-      this.unlistenFuncs_.push(listen(this.win_.document.documentElement,
+      this.unlistenFuncs_.push(listen(this.win_.document,
         ACTIVE_EVENT_TYPES[i], this.boundHandleActivity_));
     }
 
     this.viewer_.onVisibilityChanged(this.boundHandleVisibilityChange_);
+    this.viewport_.onScroll(this.boundHandleActivity_);
```

Comment:
> That's a bug, want to fix it to?

I might but maybe not in this PR if possible. At this point the unlisten functions in `Activity` are never called since `cleanup_` is never called. That means it doesn't make much of a difference for this change.